src/tests/roc_sndio/target_sox/test_sox_source.cpp

```diff
@@ -31,6 +31,9 @@ enum {
     NumChans = 2
 };
 
+const core::nanoseconds_t MaxBufDuration =
+    MaxBufSize * core::Second / (SampleRate * packet::num_channels(ChMask));
```

Comment:
I guess in this test we need FrameDuration (calculated from FrameSize) instead of MaxBufDuration (calculated from MaxBufSize).

Reply:
done.